### DISCUSSION TITLE
how fix to Projact

### DISCUSSION THREAD

**Discussion by AbdalaMask** [unoplatform/uno#5619] [Q&A]
Uwp
![image](https://user-images.githubusercontent.com/37013638/112890337-0d462000-90d7-11eb-8def-3405417f85eb.png)

to wpf 
![image](https://user-images.githubusercontent.com/37013638/112890657-6a41d600-90d7-11eb-836a-87b5293dcd94.png)

Android

**Comment by AbdalaMask** [comment]
![image](https://user-images.githubusercontent.com/37013638/112893230-b0e4ff80-90da-11eb-87a3-c1490d8c64fc.png)

**Comment by AbdalaMask** [comment]
web using long time to open 
![image](https://user-images.githubusercontent.com/37013638/112894198-db838800-90db-11eb-93a4-c79c2941d051.png)

**Comment by AbdalaMask** [comment]
How to fix colors and animation in other players

**Reply by jeromelaban** [reply]
It's difficult to help you there, we do not have enough information. What's not working ?

**Reply by AbdalaMask** [reply]
WPF

**Reply by AbdalaMask** [reply]
web using long time to open

**Reply by AbdalaMask** [reply]
How to fix colors and animation

**Comment by AbdalaMask** [comment]
<Rectangle x:Name="xBackground" Canvas.Left="5" Canvas.Top="5" Height="30" Width="90" StrokeThickness="0.5" Stroke="Black" RadiusX="3" RadiusY="3">
                              <Rectangle.Fill>
                                    <media:RadialGradientBrush Center="0.6,0.35" GradientOrigin="0.6,0.35" RadiusY="0.67" RadiusX="0.67">
                                        <GradientStop Color="{Binding OnColor, Converter={StaticResource ColorLightConverter1}, ConverterParameter=50, RelativeSource={RelativeSource Mode=TemplatedParent}}" Offset="0" />
                                        <GradientStop Color="{Binding OnColor, RelativeSource={RelativeSource Mode=TemplatedParent}}" Offset="1" />
                                    </media:RadialGradientBrush>
                                </Rectangle.Fill>
                            </Rectangle>
not working

**Comment by AbdalaMask** [comment]
<Rectangle x:Name="xBackground" Canvas.Left="5" Canvas.Top="5" Height="30" Width="90" StrokeThickness="0.5" Stroke="Black" RadiusX="3" RadiusY="3">
                              <Rectangle.Fill>
                                    <media:RadialGradientBrush Center="0.6,0.35" GradientOrigin="0.6,0.35" RadiusY="0.67" RadiusX="0.67">
                                        <GradientStop Color="{Binding OnColor, Converter={StaticResource ColorLightConverter1}, ConverterParameter=50, RelativeSource={RelativeSource Mode=TemplatedParent}}" Offset="0" />
                                        <GradientStop Color="{Binding OnColor, RelativeSource={RelativeSource Mode=TemplatedParent}}" Offset="1" />
                                    </media:RadialGradientBrush>
                                </Rectangle.Fill>
                            </Rectangle>

**Comment by AbdalaMask** [comment]
` <Rectangle x:Name="xBackground" Canvas.Left="5" Canvas.Top="5" Height="30" Width="90" StrokeThickness="0.5" Stroke="Black" RadiusX="3" RadiusY="3">
                              <Rectangle.Fill>
                                    <media:RadialGradientBrush Center="0.6,0.35" GradientOrigin="0.6,0.35" RadiusY="0.67" RadiusX="0.67">
                                        <GradientStop Color="{Binding OnColor, Converter={StaticResource ColorLightConverter1}, ConverterParameter=50, RelativeSource={RelativeSource Mode=TemplatedParent}}" Offset="0" />
                                        <GradientStop Color="{Binding OnColor, RelativeSource={RelativeSource Mode=TemplatedParent}}" Offset="1" />
                                    </media:RadialGradientBrush>
                                </Rectangle.Fill>
                            </Rectangle>`

**Comment by AbdalaMask** [comment]
No one answered my questions

**Reply by AbdalaMask** [reply]
Requested registry access is not allowed

**Reply by jeromelaban** [reply]
I'm sorry @AbdalaMask, but it's difficult for us to answer your questions, there are not enough information to do so.

If you can, try creating a sample application from the uno sample template, make it the simplest as possible to show the issue and post it here.

**Reply by AbdalaMask** [reply]
thank you for replying

**Reply by AbdalaMask** [reply]
https://github.com/AbdalaMask/EVStudioScadaFull_FromUno

**Reply by AbdalaMask** [reply]
Please see the project, respond and show the error
In it

**Reply by AbdalaMask** [reply]
> I'm sorry @AbdalaMask, but it's difficult for us to answer your questions, there are not enough information to do so.
> 
> If you can, try creating a sample application from the uno sample template, make it the simplest as possible to show the issue and post it here.

https://github.com/AbdalaMask/EVStudioScadaFull_FromUno
Please see the project, respond and show the error
In it

**Comment by AbdalaMask** [comment]
The SCAD program works on Wpf
And I want it to work on all systems
And your library is excellent for that
But when there are some problems
I analyzed some of them
But the colors and animation
It does not work well as in the attached pictures

**Comment by AbdalaMask** [comment]
![image](https://user-images.githubusercontent.com/37013638/113437231-9070ba00-93e6-11eb-9cdb-e85c1b3e4752.png)
work to Uwp 
not work to Web and Andrud
